### PR TITLE
[tools][doc]add ssb flat tools, and update performance doc

### PR DESCRIPTION
**Pull request by hello-stephen**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

add some scripts for SSB flat test and update the performance doc, which is in order to make the test more easily.
I only write the Chinese version as the base.

## Checklist(Required)

1. Does it affect the original behavior: (No)
2. Has unit tests been added: (No)
3. Has document been added or modified: (Yes)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by morningman — DISMISSED** [review]
LGTM

File: docs/zh-CN/benchmark/ssb-and-flat.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
+---
```
Comment (EmmyMiao87) [inline docs/zh-CN/benchmark/ssb-and-flat.md:1]: Please add this doc in sidebar

File: docs/zh-CN/benchmark/ssb-and-flat.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+{
+    "title": "Star-Schema-Benchmark 测试",
+    "language": "zh-CN"
```
Comment (EmmyMiao87) [inline docs/zh-CN/benchmark/ssb-and-flat.md:4]: Why do Chinese and English SSB test documents not correspond?

Comment (hello-stephen) [reply docs/zh-CN/benchmark/ssb-and-flat.md:4]: I just provide a basic version, and Jiafeng will improve the documentation next. This document may be displayed to users, and Jiafeng is better at writing documents.

**Review by hf200012 — DISMISSED** [review]
LGTM

**Comment by hf200012** [comment]
@morningman @EmmyMiao87 I have already told @hello-stephen  about this compilation error. After passing it, I will merge it in, and I will improve this document. There are still some places to be modified and the English document to be improved.

**Review by hf200012 — DISMISSED** [review]
LGTM

**Review by hf200012 — DISMISSED** [review]
LGTM

File: docs/en/benchmark/ssb.md [hunk 3]
```diff
@@ -26,173 +26,363 @@ under the License.
...
+> Note 1: The standard test set including SSB is usually far from the actual business scenario, and some tests will perform parameter tuning for the test set. Therefore, the test results of the standard test set can only reflect the performance of the database in specific scenarios. Users are advised to conduct further testing with actual business data.
 >
-> Note 2: The operations involved in this document are all performed in the CentOS 7 environment.
+> Note 2: The operations involved in this document are all performed in the Ubuntu Server 20.04 environment, and CentOS 7 can also be tested.
 
-## Environmental preparation
+On 13 queries on the SSB standard test dataset, we tested the upcoming Doris 1.1 version and Doris 0.15.3 version peer-to-peer, and the overall performance improved by 2-3 times.
 
-Please refer to the [official document](../install/install-deploy.md) to install and deploy Doris to obtain a normal running Doris cluster ( Contain at least 1 FE, 1 BE).
+![ssb_v11_v015_compare](images/ssb_v11_v015_compare.png)
```
Comment (hf200012) [inline docs/en/benchmark/ssb.md]:
```suggestion
![ssb_v11_v015_compare](/images/ssb_v11_v015_compare.png)
```

**Review by hf200012 — DISMISSED** [review]
LGTM

**Review by hf200012 — APPROVED** [review]
LGTM

**Pull request merged**